usaspending_api/agency/v2/views/budgetary_resources.py

```diff
@@ -18,82 +16,58 @@ class BudgetaryResources(AgencyBase):
 
     @cache_response()
     def get(self, request, *args, **kwargs):
-        agency_budgetary_resources, agency_total_obligated = self.get_agency_budgetary_resources()
         return Response(
             {
-                "fiscal_year": self.fiscal_year,
                 "toptier_code": self.toptier_agency.toptier_code,
-                "agency_budgetary_resources": agency_budgetary_resources,
-                "prior_year_agency_budgetary_resources": self.get_prior_year_agency_budgetary_resources(),
-                "total_federal_budgetary_resources": self.get_total_federal_budgetary_resources(),
-                "agency_total_obligated": agency_total_obligated,
-                "agency_obligation_by_period": self.get_agency_obligations_by_period(),
+                "agency_data_by_year": self.get_agency_budgetary_resources(),
                 "messages": self.standard_response_messages,
             }
         )
 
-    def get_total_federal_budgetary_resources(self):
+    def get_total_federal_budgetary_resources(self, fiscal_year):
         """
         IMPORTANT NOTE!  This is placeholder functionality.  DEV-4014 addresses the underlying problem of
         not having actual historical budgetary resources.  This code is here to provide semi-reasonable
         values for development and testing until such time as legitimate values are available.  A note
         has been added to DEV-4014 to rectify this once that ticket has been resolved.
         """
-        return AppropriationAccountBalances.objects.filter(
-            submission__reporting_fiscal_year=self.fiscal_year, submission__reporting_fiscal_period=self.fiscal_period
-        ).aggregate(total_federal_budgetary_resources=Sum("total_budgetary_resources_amount_cpe"))[
-            "total_federal_budgetary_resources"
-        ]
+        return AppropriationAccountBalances.objects.filter(submission__reporting_fiscal_year=fiscal_year).aggregate(
+            total_federal_budgetary_resources=Sum("total_budgetary_resources_amount_cpe")
+        )["total_federal_budgetary_resources"]
```

Comment:
As the comment describes, this functionality is not the correct solution. What has changed since then is this endpoint doesn't require the static value to be updated, that can be ignored. Total Budgetary Resources should be obtained from GTAS. I recommend a single SQL query instead of creating this to make a SQL call of each fiscal year.